components/02-elements/list/_list.scss

```diff
@@ -32,6 +32,22 @@ $list-divider            : 1px solid $gray-dark !default;
         }
     }
 
+    &__label {
+        display: inline-block;
+        clear: left;
+        float: left;
```

Comment:
`float: left` with `display: inline-block` makes no sense at all. Also we should avoid using `floats` and `inline-block` too.